docs/config/index.md

```diff
@@ -849,3 +849,58 @@ Vitest usually uses cache to sort tests, so long running tests start earlier - t
 - **Default**: `Date.now()`
 
 Sets the randomization seed, if tests are running in random order.
+
+### typechecker
```

Comment:
```suggestion
### typecheck
```

Seems a typo?

Reply:
And on `docs/guide/testing-types.md` page